### DISCUSSION TITLE
Hydrogen Cyanide in biosphere database

### DISCUSSION THREAD

**Discussion by yviph** [LCA-ActivityBrowser/activity-browser#977] [Q&A]
Hi all,

I am looking for Hydrogen Cyanide in the biosphere database...
I have tried several other names for HCN, but couldn't find it, is it possible that it is missing in the biosphere database of AB?

Cheers, Yvonne

**Comment by Zoophobus** [comment]
Hi Yvonne,

it seems you are right that Hydrogen Cyanide is not present in the biosphere databases (versions 3.8, or 3.9). Could you use Cyanide as a proxy, or would you need to create your own Hydrogen Cyanide biosphere process?

**Reply by yviph** [reply]
Thanks for checking!
I would like to avoid using Cyanide as Proxy and was about to check to create my own HCN biosphere flow, but noticed that there are no Chracterisation Factors available in ReCiPe... so if anybody has a hint about CF for HCN, please let me know :)

**Reply by yviph** [reply]
And I am also not sure if I can create an own biosphere emission flow, I just managed to create a product flow...
